### ISSUE TITLE
Failed to find english language item : ' + e.name

### ISSUE THREAD

**Issue by enmohsinali** [fawazahmed0/youtube-uploader#134]
```/home/projects/youtube/node_modules/youtube-videos-uploader/dist/upload.js:641
            throw new Error('Failed to find english language item : ' + e.name);
                  ^

Error: Failed to find english language item : TimeoutError
    at /home/projects/youtube/node_modules/youtube-videos-uploader/dist/upload.js:641:19
    at Generator.throw (<anonymous>)
    at rejected (/home/projects/youtube/node_modules/youtube-videos-uploader/dist/upload.js:6:65)
```

**Comment by fawazahmed0** [comment]
If someone tests that PR and confirms everything is working fine, then I will be happy to merge it. Otherwise I won't merge it.

**Comment by DominicFindlay** [comment]
When I tried #133 it worked, as in no error, and starts uploading.


(This probs should be a new Issue...)
However was stuck waiting for a long time after uploading which I think is due to the most recent changes changing the text looked for when "Upload complete" vs "Video upload complete"
I have the skipProcessingWait flag set to true but it's first waiting for "Upload complete", changing it to "Video upload complete" appears to work for me.

**Comment by fawazahmed0** [comment]
Please see https://github.com/fawazahmed0/youtube-uploader/issues/131 , using "Video upload complete" causes problems to others

**Comment by pc11000** [comment]
Same error here:

`Error: Failed to find english language item : TimeoutError`

any updates how to fix it?

**Comment by cip78** [comment]
Hi, can anyone put the whole upload.ts file with modifications to skip error? I changed it but i cant make it work. Ty

**Comment by fawazahmed0** [comment]
I have merged #133 , please try v1.3.64

**Comment by cip78** [comment]
> I have merged #133 , please try v1.3.64

ty. That part works now, but its stuck at completing the Title, description..etc.

**Comment by cip78** [comment]
Can someone make it work for me, i willing to pay on freelancer platform.

**Comment by fawazahmed0** [comment]
write email to [youtube-uploader@datascraping.pp.ua](mailto:youtube-uploader@datascraping.pp.ua)

**Comment by fawazahmed0** [comment]
This issue should be fixed in latest version v1.3.66 , thanks to @cip78 for getting this fixed
